test/fast/Unit tests/nvm_command_info

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+cleanup() {
+  unalias wget > /dev/null 2>&1
+  unset -f wget > /dev/null 2>&1
+  unset WGET_EXPECTED_INFO WGET_COMMAND_INFO
+}
+
+die() { cleanup; echo "$@" ; exit 1; }
+
+\. ../../../nvm.sh
+
+
+# 1. test wget command
+WGET_COMMAND_INFO="$(nvm_command_info wget)"
+WGET_EXPECTED_INFO="$(which wget)"
+[ "${WGET_COMMAND_INFO}" = "${WGET_EXPECTED_INFO}" ] || die "wget command info wrong(stage 1), expected: '${WGET_EXPECTED_INFO}', got '${WGET_COMMAND_INFO}'"
+
+# clean up this stage
+unset WGET_EXPECTED_INFO WGET_COMMAND_INFO
+
+
+# 2. test aliased wget
+shopt -s expand_aliases
+# enable expand_aliases to make alias working in interactive shell
+alias wget="wget -V"
+WGET_COMMAND_INFO="$(nvm_command_info wget)"
+WGET_EXPECTED_INFO="$(which wget) (wget -V)"
+[ "${WGET_COMMAND_INFO}" = "${WGET_EXPECTED_INFO}" ] || die "wget command info wrong(stage 2), expected: '${WGET_EXPECTED_INFO}', got '${WGET_COMMAND_INFO}'"
+
+# clean up this stage
+unalias wget
```

Comment:
I think we can treat the `cleanup()` as the final process to do before exit but before that the cleanup process is a little bit different?